path_dynamic_creds.go

```diff
@@ -15,6 +16,30 @@ import (
 	"golang.org/x/text/encoding/unicode"
 )
 
+func (b *backend) pathDynamicCredsCreate() []*framework.Path {
+	return []*framework.Path{
+		{
+			Pattern: path.Join(dynamicCredPath, framework.MatchAllRegex("name")),
+			Fields: map[string]*framework.FieldSchema{
+				"name": {
+					Type:        framework.TypeLowerCaseString,
+					Description: "Name of the dynamic role.",
+				},
+			},
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: b.pathDynamicCredsRead,
+				},
+			},
+			HelpSynopsis: "Request LDAP credentials for a dynamic role. These credentials are " +
+				"created within the LDAP system when querying this endpoint.",
+			HelpDescription: "This path requests new LDAP credentials for a certain dynamic role. " +
+				"The credentials are created within the LDAP system based on the creation_ldif " +
+				"specified within the dynamic role configuration.",
```

Comment:
Maybe worth adding a note about eventual consistency?

Reply:
Added a note in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/40/commits/8911ba44f573b226dff33f490ca13f016054686f 👍